packages/frontend/pages/_app.tsx

```diff
@@ -46,13 +51,26 @@ function MyApp({ Component, pageProps }: AppProps) {
     }
   }, [address, fetchPositions, updateVault]);
 
+  useEffect(() => {
+    if (address) {
+      pollBalances();
+    } else {
+      stopPolling();
+    }
+    return () => {
+      stopPolling();
+    };
+  }, [address]);
+
   useEffect(() => {
     const handleRouteChange = (url: string) => {
       const isTop = isTopLevelUrl(url);
       if (isTop && address) {
         fetchPositions();
         updateVault();
       }
+      const should = url === '/borrow' || url.includes('/my-positions/');
```

Comment:
Isn't it better to have all urls as constants in a separate file so that if we want to change them, to have them in a centralized manner?